docs/docs/schema-connections.md

```diff
@@ -2,6 +2,10 @@
 title: Schema connections
 ---
 
+> This documentation isn't up to date with latest
```

Comment:
Is there an issue open for updating the Schema Connections doc? It would help to list what needs to be updated so people are pointed in the right direction (and having an issue would communicate the need for it even more, as people look for good issues to pick up)

Reply:
I've added a link to an issue for rewriting the Gatsby Internals docs for schema related stuff.